src/components/MAPI/workernodes/ClusterDetailWorkerNodes.tsx

```diff
@@ -382,9 +375,11 @@ const ClusterDetailWorkerNodes: React.FC<
       );
     }, [nodePoolList?.items, providerNodePools]);
 
-    const clusterReleaseVersion = cluster
-      ? capiv1beta1.getReleaseVersion(cluster)
-      : undefined;
+    const isReleasesSupportedByProvider = supportsReleases(provider);
```

Comment:
We fetch releases to determine node pool cgroups versions, but if Release CRs are not supported by the provider, we should refrain from making the request to fetch them altogether.